### DISCUSSION TITLE
Droppable area not calling onDragOver when empty

### DISCUSSION THREAD

**Discussion by mwillis2300** [clauderic/dnd-kit#1130] [Q&A] [answered]
Trying to do email app with TO and CC fields. Want ability to move email chips around. For example, dragging an email chip from TO field to CC field. The problem is that if a field is empty, dragging over that area no longer triggers onDragOver. 

A question like this has been asked before and the answer was to use useDroppable. I'm using that for both TO and CC fields with no luck. 

https://codesandbox.io/s/dnd-kit-multi-horizontal-sortable-mzgxfu?file=%2Fsrc%2F

Thanks!

**Comment by blnvdanil** [comment, marked as answer]
This seems to be related to your problem: https://github.com/clauderic/dnd-kit/issues/432

**Reply by mwillis2300** [reply]
Yes, that was it! Changing collisionDetection to rectIntersection triggered onDragOver when the container was empty. Thanks so much for your help!
